pkg/mdformatter/linktransformer/link.go

```diff
@@ -239,6 +251,15 @@ func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []
 	if config.Timeout != "" {
 		v.c.SetRequestTimeout(config.timeout)
 	}
+
+	if v.validateConfig.CacheType == sqlite && storage != nil {
```

Comment:
Do we need the cache type check? :thinking: As long as `storage` is not `nil`, it should not matter what type we're initializing, right?

Reply:
Hm then I would maybe expect `storage` to still be `nil`, after all if we do not want to initialize storage why would it not be `nil`? I think it's detail, it just feels like it could be simplified a bit, but it's more of a nit.

Reply:
still to be done?

Reply:
We can iterate on this in separate PR!